examples/connext_dds/dynamic_data_request_reply/README.md

```diff
@@ -0,0 +1,79 @@
+# Example Code: Request-Reply DynamicData Application
+
+## Concept
+
+*Dynamic Data* API allows to create topic samples in a programmatically manner
+without defining an IDL in compile time.
+
+*Dynamic Data* provides an easy way to access the members of a data sample. Such
+members can be primitive data types or complex data types such as arrays or
+sequences.
+
+While the access to certain members is done using a name, the access in multiple
```

Comment:
Is this related to the example or to the Dynamic Data API? Dynamic Data API also allows accessing members based on it's ID.

Reply:
yes, this is related to the Dynamic Data API, should I add something like "...using a name or ID."?